src/AspNet.Security.OpenId/OpenIdAuthenticationInitializer.cs

```diff
@@ -17,21 +17,14 @@ namespace Microsoft.Extensions.DependencyInjection;
 /// Contains the methods required to ensure that the configuration used by
 /// the OpenID 2.0 generic handler is in a consistent and valid state.
 /// </summary>
+/// <remarks>
```

Comment:
Hum, that looks weird. Is it suggested by a Roslyn code fixer?

Reply:
Yeah the code-fixer did that. I _assumed_ it was correct and is the way you document a constructor if using primary constructors.

Reply:
If this is wrong, let me know and I'll update post-merge.

Reply:
Looks too weird to my taste, let's remove these <remarks> nodes, they don't have anything super useful anyway 😄